test/acceptance/service-fabrik-admin.instances.director.spec.js

```diff
@@ -119,6 +119,30 @@ describe('service-fabrik-admin', function () {
               mocks.verify();
             });
         });
+        it('should initiate a service-fabrik-(update)operation with forbidden manifest changes disable', function () {
+          mocks.cloudController.findServicePlan(instance_id, plan_unique_id);
+          mocks.director.getDeploymentManifest();
+          mocks.director.diffDeploymentManifest();
+          mocks.cloudController.updateServiceInstance(instance_id, body => {
+            const token = _.get(body.parameters, 'service-fabrik-operation');
+            return support.jwt.verify(token, name, args);
+          });
+          return chai
+            .request(apps.internal)
+            .post(`${base_url}/deployments/${deployment_name}/update`)
+            .send({
+              forbidden_changes: 'false'
```

Comment:
why pass boolean as string ?

Reply:
was mimicking curl behaviour.

Reply:
Fixed this in code using JSON.parse() now works for both string and boolean.

Reply:
ok , got it.